test/blackbox-tests/test-cases/melange/installed-library-with-runtime-deps.t

```diff
@@ -0,0 +1,80 @@
+Test simple interactions between melange.emit and copy_files
```

Comment:
I think this should be

```suggestion
Test simple interactions between melange.emit and runtime_deps
```

? Same for other 2 tests.

Reply:
Done in ef9926dbc